### PR TITLE
Fix warnings in back-ends

### PR DESCRIPTION
**Pull request by nabijaczleweli**
Also make BLT 80x40 because h=80 is fucking gigantic and doesn't fit on-screen, would be good to make it configurable somehow?

### CODE REVIEW

**Comment by gyscos** [comment]
I think the default terminal size is historically 80x24.
A configurable value would be very nice, but I'm not sure how to deal with backend specific configuration. The easiest way could be to try to read a hardcoded filename, but that's not very flexible. Maybe compile-time environment variables?...
Is like to avoid having backend-specific user code, but maybe it can't cleanly be done without.

File: src/backend/curses/mod.rs [hunk 1]
```diff
@@ -11,6 +12,7 @@ mod pan;
 pub use self::pan::*;
 
 
+#[cfg(any(feature = "ncurses", feature = "pancurses"))]
```
Comment (gyscos) [inline src/backend/curses/mod.rs]: Maybe put those in `backend/mod.rs` instead? That way we'd only need one.

**Comment by gyscos** [comment]
Making the window resizeable would already be a boost in user control actually.

File: src/backend/curses/mod.rs (hunk 1 above)
Comment (nabijaczleweli) [inline src/backend/curses/mod.rs]: Addressed

**Pull request merged**